File: app/src/main/java/com/BlizzardArmory/model/warcraft/charactersummary/Media.java
package com.BlizzardArmory.model.warcraft.charactersummary;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * The type Media.
 */
public class Media {

    @SerializedName("href")
    @Expose
    private String href;

    /**
     * Gets href.
     *
     * @return the href
     */
    public String getHref() {
        return href;
    }

    /**
     * Sets href.
     *
     * @param href the href
     */
    public void setHref(String href) {
        this.href = href;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/equipment/Media.kt
package com.BlizzardArmory.model.warcraft.equipment

import com.google.gson.annotations.SerializedName


/**
 * The type Media.
 */
data class Media(

        @SerializedName("key")
        var key: Key,

        @SerializedName("id")
        var id: Long

)